src/main/java/formflow/library/FormFlowController.java

```diff
@@ -19,12 +19,16 @@
 @Slf4j
 public abstract class FormFlowController {
 
+  /**
+   * This is an abstract controller class used to pass services and helper methods to controllers.
```

Comment:
This is probably better described as the parent class to the other controllers.  They inherit from this and the have access to all these variables and methods.

Reply:
done

Reply:
done